flutter/lib/src/flutter_enricher.dart

```diff
@@ -0,0 +1,123 @@
+import 'dart:async';
+import 'dart:ui';
+
+import 'package:device_info_plus/device_info_plus.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:sentry/sentry.dart';
+
+/// Enriches [SentryEvent]s with various kinds of information.
+/// FlutterEnricher only needs to add information which aren't exposed by
+/// the Dart runtime.
+class FlutterEnricher implements Enricher {
+  static final Enricher instance = FlutterEnricher(
+      PlatformChecker(),
+      DeviceInfoPlugin(),
+      Enricher.defaultEnricher,
+      WidgetsFlutterBinding.ensureInitialized().window);
+
+  FlutterEnricher(
+    this._checker,
+    this._deviceInfoPlugin,
+    this._dartEnricher,
+    this._window,
+  );
+
+  final SingletonFlutterWindow _window;
+  final PlatformChecker _checker;
+  final DeviceInfoPlugin _deviceInfoPlugin;
+  final Enricher _dartEnricher;
+  WindowsDeviceInfo? _windowsDeviceInfo;
+  LinuxDeviceInfo? _linuxDeviceInfo;
+
+  @override
+  FutureOr<SentryEvent> apply(SentryEvent event) async {
+    if (_checker.hasNativeIntegration) {
```

Comment:
Even for platforms with native integrations this should be executed  for those Flutter & Dart specific informations